README.rst

```diff
@@ -59,11 +59,11 @@ Configuration Options (.ini)
 
 Can accept `Esri basemap name <http://esri.github.io/esri-leaflet/api-reference/layers/basemap-layer.html>`_ or generic tile url template::
 
-  ckanext.agsview.default_basemap_url = Gray
+  ckanext.ags_view_default_basemap_url = Gray
```

Comment:
I think the other example of the ckanext.ags_view_default_basemap_url should be here too.
```
or
ckanext.ags_view_default_basemap_url = http://example.com/MapServer/tile/{z}/{x}/{y}
```

Reply:
I agree, and there is an example of the `default_basemap_url` in the above `ArcGIS Feature Layer Viewer (ags_fs_view)` section. This change is for the ini file.